_grades/A9.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+title: A9 - Lead Software Engineer
+permalink: /grades/A9
+---
+
+## Technical Skills (Dex)
+
+Despite being less hands-on, you can instantly pick up and understand technical problems, recommend debugging tactics, and choose a solution from options presented
+
+When presented with a number of solutions, you can quickly and confidently select the best option to allow your team to progress
+
+You can predict market trends and steer your team through architectural changes.
+
+You frequently attend technical conferences, absorb and share key messages across your teams
+
+Project software architects come to you for input
+
+## Soft Skills (T)
+
+You appreciate, and take into account, other teams’ perspective on technical projects (PMO, Operations etc). You have a good working relationship with these teams and they come to you for advice.
+
+You can gauge the wellbeing of your team and constantly work towards improving it
+
+You are a confident public speaker in your chosen domain, you can clearly articulate your thoughts
+
+## Getting Code Live (Str)
+
+You are able to provide advice on estimation in multiple environments
+
+You can ask the right questions to enable a team to generate confidence in their deployment process
+
+You have the right contacts in other teams to get help troubleshooting deployment issues on multiple platforms
+
+## Impact (Wis)
+
+### Solution Ownership
+
+You have been responsible for one or more large project releases
```

Comment:
Is it worth quantifying this a bit? What would a large project be, maybe one that involves multiple development teams?

Reply:
@sasaunde What are your views on the above?

Reply:
Good point it is a bit vague. Maybe "you have made technical decisions affecting a large project of multiple scrum teams"?